pennylane/math/quantum.py

```diff
@@ -938,3 +938,47 @@ def _check_state_vector(state_vector):
     if not is_abstract(norm):
         if not allclose(norm, 1.0, atol=1e-10):
             raise ValueError("Sum of amplitudes-squared does not equal one.")
+
+
+def min_entropy(state, base=None, check_state=False, c_dtype="complex128"):

```

Comment:
Looking good so far @devilkiller-ag 💯

I'll do a full review once the tests for this function are written.

Reply:
Great, I will complete writing tests soon. Thanks for the review